docs/src/main/paradox/docs/delta/metadata.md

```diff
@@ -0,0 +1,30 @@
+# Nexus Metadata
+
+Nexus Delta entities fall in two categories:
+
+- Global: @ref:[Organizations](api/orgs-api.md), @ref:[Realms](api/realms-api.md),
+  @ref:[Permissions](api/permissions-api.md), @ref[ACLs](api/acls-api.md)
+- Project scoped: @ref:[Projects](api/projects-api.md), @ref:[Resources](api/resources-api.md),
+  @ref:[Schemas](api/schemas-api.md), @ref:[Resolvers](api/resolvers-api.md),
+  @ref:[Views](api/views/index.md), @ref:[Storages](api/storages-api.md), @ref:[Files](api/files-api.md)
+
+Upon creation of these entities, Nexus Delta will create metadata fields which are described below.
+
+## Identification
```

Comment:
Navigation Link
- Mentioning HATEOAS + link to the wk page
- Remove mention to identification
- Saying this endpoint allows to fetch the entity (for example, from  listing operations)